### PR TITLE
[PB-424]: (feature) New migration flow UI

### PR DESCRIPTION
**Pull request by PixoDev**
Migration flow UI, the system that performs the actual migration is not included in this PR, a fake migration for demo purposes has been added here.

Translations for EN, ES and FR are included

### CODE REVIEW

File: src/main/device/service.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import path from 'path';
 import { getHeaders } from '../auth/service';
 import { addGeneralIssue } from '../background-processes/process-issues';
 import configStore from '../config';
+import { getVirtualDrivePath } from 'workers/webdav/VirtualDrive';
```
Comment (larry-internxt) [inline src/main/device/service.ts:11]: i think this new imports are not being used

File: src/main/sync-root-folder/handlers.ts [hunk 2]
```diff
@@ -9,8 +9,10 @@ ipcMain.handle('get-sync-root', () => {
...
 ipcMain.handle('choose-sync-root-with-dialog', chooseSyncRootWithDialog);
 
-ipcMain.handle('open-sync-folder', () => {
+ipcMain.handle('open-sync-folder', async () => {
   const syncFolderPath = configStore.get('syncRoot');
 
-  return shell.openPath(syncFolderPath);
+  const errorMessage = await shell.openPath(syncFolderPath);
+
+  if (errorMessage) throw new Error(errorMessage);
```
Comment (JoanVicens) [inline src/main/sync-root-folder/handlers.ts:17]: We cannot throw errors on the handlers https://github.com/electron/electron/issues/24427

File: src/renderer/pages/Onboarding/index.tsx [hunk 3]
```diff
@@ -13,6 +13,11 @@ export default function Onboarding() {
   const [backupFolders, setBackupFolders] = useState<BackupFolder[]>([]);
   const [slideIndex, setSlideIndex] = useState<number>(0);
   const [backupsModalOpen, setBackupsModalOpen] = useState(false);
+  const [platform, setPlatform] = useState<string>('');
```
Comment (JoanVicens) [inline src/renderer/pages/Onboarding/index.tsx:16]: ℹ️  There is a hook to get the platform `useClientPlatform`

File: src/renderer/pages/Onboarding/slides/AvailableOnlineSlide.tsx [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import { OnboardingSlideProps } from '../helpers';
+
 import { useTranslationContext } from 'renderer/context/LocalContext';
 
-export type AvailableOnlineSlideProps = OnboardingSlideProps;
+export type AvailableOnlineSlideProps = object;
```
Comment (JoanVicens) [inline src/renderer/pages/Onboarding/slides/AvailableOnlineSlide.tsx:5]: If the props are of type `object` do we need them? I guess this is for the demo

File: src/renderer/pages/Onboarding/index.tsx (hunk 3 above)
Comment (PixoDev) [reply src/renderer/pages/Onboarding/index.tsx:16]: glorious, thx didn't see it

File: src/renderer/pages/Onboarding/slides/AvailableOnlineSlide.tsx (hunk 4 above)
Comment (PixoDev) [reply src/renderer/pages/Onboarding/slides/AvailableOnlineSlide.tsx:5]: I think will be better to not use props here yeah

**Pull request merged**